lib/types/instagram/parse.js

```diff
@@ -2,6 +2,39 @@ import last from 'lodash.last';
 
 const type = 'instagram';
 
+const getUserElm = container => {
+  let elm = container.childNodes[0];
+  if (elm && elm.childNodes) {
+    elm = elm.childNodes[0];
+  }
+
+  return (elm && elm.nodeName === '#text') ? elm : null;
+};
+
+const getUser = elm => {
+  const userElm = getUserElm(elm);
+  if (!userElm) {
+    return {name: null, slug: null};
+  }
+  const userString = userElm.data;
+  const lastIndex = userString.lastIndexOf('(');
+  const name = userString.slice(2, lastIndex).replace('photo posted by', '').trim();
+  const slug = userString.slice(lastIndex + 2, userString.lastIndexOf(')'));
+
+  return {name, slug};
+};
+
+const getDate = elm => {
+  const time = elm && elm.getElementsByTagName('time')[0];
+  if (!time) {
+    return {urc: null, string: null};
```

Comment:
@iefserge yeah, I'd love it if standard was more strict around that. I've gathered that you (and some others at Mic) prefer `{foo: 'bar'}` over `{ foo: 'bar' }` but I don't really have any preference. I'll go through the code and make sure I'm consistent (`{foo: 'bar'}` - thanks!